constants.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    htmlSeparatorsDefault: {
```

Comment:
can we put this in the code like before? It is a little silly to bring it out of the file and then do an assignment. It is easier to read if we don't have this.